pkgs/applications/audio/mamba/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, xorg, cairo, libjack2, pcre, libXdmcp, libsigcxx, liblo, pkgconfig }:
```

Comment:
You can also format the input function like so:
{ stdenv
, fetchFromGitHub
, ...
}:

This makes future diffs more readable.

Reply:
All don, thanks for the feedback!